bindings/cpp/src/android.cc

```diff
@@ -0,0 +1,101 @@
+#include <jni.h>
+#include <string>
+#include "defi-wallet-core-cpp/include/android.h"
+#define SECURE_STORAGE_CLASS "com/cronos/play/SecureStorage"
+using namespace std;
+
+JNIEnv *g_env = NULL;
+namespace org {
+namespace defi_wallet_core {
+
+int secureStorageSetJavaEnv(JNIEnv *userenv) {
```

Comment:
what is the function used for? Why it return 1, looks twicky.

Reply:
to call java api, need jnienv pointer, this exists for android application.